client/landing/jetpack-cloud/components/sidebar/index.jsx

```diff
@@ -69,14 +70,16 @@ class JetpackCloudSidebar extends Component {
 	};
 
 	render() {
+		const { translate } = this.props;
+
 		return (
 			<Sidebar>
 				<SidebarRegion>
 					{ /* @todo: A profile info box needs to be created and added here; similar to <ProfileGravatar /> in client/me/sidebar/index.jsx */ }
 					<SidebarMenu>
 						<SidebarItem
 							link="/"
-							label="Dashboard" // @todo: Localize
+							label={ translate( 'Dashboard' ) }
```

Comment:
It would make sense to add more context information for the translators?, such as `translate( 'Dashboard' ), {
		context: 'Navigation item',
		textOnly: true
} `

The same for the rest of the Sidebar items.

Reply:
Good idea! I'll take care of that in the next commit.

Reply:
This has been addressed in f0b2574

Note that according to the [i18n-calypso docs](https://github.com/Automattic/wp-calypso/blob/master/packages/i18n-calypso/README.md#translate-method), there is no `textOnly` option available in `translate`. Also, I've used `comment` instead of `context` as I think it's more suitable.

Reply:
Great! I extracted the sample from https://github.com/Automattic/wp-calypso/blob/master/client/my-sites/plugins/plugin-sections/index.jsx